julia/cxx/bb.cxx

```diff
@@ -0,0 +1,173 @@
+// CHORD 8-bit baseband beamformer
+// See <https://www.overleaf.com/project/6228adae742a3a2da1afe437>
```

Comment:
Maybe reference the internally linked PDF, since most people won't have access to the overleaf?

Reply:
Good idea, done